modules/cas_cache/layer_cache_management.c

```diff
@@ -1894,6 +1896,9 @@ int cache_mngt_init_instance(struct ocf_mngt_cache_config *cfg,
 	if (result != -KCAS_ERR_WAITING_INTERRUPTED)
 		kfree(context);
 
+	if (!result)
+		cache_priv->ctx = NULL;
```

Comment:
is this needed?

Reply:
Not needed, but at this point, `ctx` would point to freed memory, so it is I would call it nice to have.